docs/docker.rst

```diff
@@ -0,0 +1,185 @@
+..
+    This file is part of INSPIRE.
+    Copyright (C) 2017 CERN.
+
+    INSPIRE is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    INSPIRE is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+
+    In applying this licence, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization
+    or submit itself to any jurisdiction.
+
+
+Docker (Linux)
+==============
+
+Docker is an application that makes it simple and easy to run processes in a container,
+which are like virtual machines, but more resource-friendly. For a detailed introduction to the
+different components of a Docker container. For a detailed introduction to the different components
+of a Docker container, you can follow `this tutorial
+<https://www.digitalocean.com/community/tutorials/the-docker-ecosystem-an-introduction-to-common-components>`_.
+
+
+Inspire and Docker
+##################
```

Comment:
Given that this entire section is copied from https://github.com/inspirehep/inspire-next/pull/1015 you should have given credit to @mihaibivol.